conda/recipes/cudf/meta.yaml

```diff
@@ -66,7 +66,7 @@ requirements:
     - pandas >=1.3,<1.6.0dev0
     - cupy >=12.0.0
     - numba >=0.56.4,<0.57
-    - numpy >=1.21
+    - numpy>=1.21,<1.24  # Temporarily upper bound numpy to avoid overflow deprecations
```

Comment:
Any reasons for removing the whitespace? TBH I don't know if it's mandatory or not, but note the pinnings above have the whitespace and we should try to keep it consistent.

Reply:
No reason other than I copy-pasted the way that it was in dependencies.yaml. We can change it back.

Reply:
Done in #13306